src/Components/Results/ProgramPage/ProgramPage.tsx

```diff
@@ -166,7 +166,11 @@ const ProgramPage = ({ program }: ProgramPageProps) => {
       </div>
       <div className="apply-button-container">
         <a className="apply-online-button" href={program.apply_button_link.default_message} target="_blank">
-          <FormattedMessage id="results.apply-online" defaultMessage="Apply Online" />
+          {program.apply_button_description.default_message == "" ? (
+            <FormattedMessage id="results.apply-online" defaultMessage="Apply Online" />
+          ) : (
+            <FormattedMessage id={program.apply_button_description.label} defaultMessage="Apply Online" />
```

Comment:
```suggestion
            <ResultsTranslate translation={program.apply_button_description} />
```

Reply:
Actually, I am going to commit this myself so we can release this today.

Reply:
Thanks for that.